### ISSUE TITLE
Error when set ftps

### ISSUE THREAD

**Issue by yotewach** [gotev/android-upload-service#459]
### What did you expect?
use FTPS with "TLS" to upload files

### Version info
* Android Upload Service version: 3.5.2
* Android version and API version: 29
* HTTP stack (e.g. HurlStack or OkHttpStack):

### Request code:

```kotlin
private fun uploadFileToServer(file: File) {
        try {
            val uploadId = UUID.randomUUID().toString()
            val request = FTPUploadRequest(this, uploadId, Config.ftpHost, Config.ftpPort)
                .useSSL(true)
                .setSecurityModeImplicit(true)
                .setSecureSocketProtocol("TLS")
                .setMaxRetries(5)
                .setNotificationConfig(getNotificationConfig(uploadId, R.string.ftp_upload))
                .setUsernameAndPassword(Config.ftpUser, Config.ftpPass)
                .setSocketTimeout(10000)
                .setConnectTimeout(10000)
                .setAutoDeleteFilesAfterSuccessfulUpload(true)

            val path = "public_html/"

            request.addFileToUpload(file.absolutePath, path)
            request.startUpload()
        } catch (e: Exception) {
            Toast.makeText(this, e.message, Toast.LENGTH_LONG).show()
        }
    }
```

#### Where have you added the request code?
- [x] Activity

### LogCat output (please [set log level to DEBUG](https://github.com/gotev/android-upload-service/wiki/Recipes#logging) first)
```
E/UploadService: UploadTask - Error in uploadId f51b6314-a522-4b53-8811-71c160f77b1c on attempt 3. Waiting 4s before next attempt. 
    java.lang.NullPointerException: Attempt to invoke virtual method 'boolean org.apache.commons.net.ftp.FTPClient.isConnected()' on a null object reference
        at net.gotev.uploadservice.ftp.FTPUploadTask.upload(FTPUploadTask.java:139)
        at net.gotev.uploadservice.UploadTask.run(UploadTask.java:162)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:764)
E/UploadService: UploadTask - Error in uploadId f51b6314-a522-4b53-8811-71c160f77b1c on attempt 4. Waiting 8s before next attempt. 
    java.lang.NullPointerException: Attempt to invoke virtual method 'boolean org.apache.commons.net.ftp.FTPClient.isConnected()' on a null object reference
        at net.gotev.uploadservice.ftp.FTPUploadTask.upload(FTPUploadTask.java:139)
        at net.gotev.uploadservice.UploadTask.run(UploadTask.java:162)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:764)
E/UploadService: UploadTask - Error in uploadId f51b6314-a522-4b53-8811-71c160f77b1c on attempt 5. Waiting 16s before next attempt. 
    java.lang.NullPointerException: Attempt to invoke virtual method 'boolean org.apache.commons.net.ftp.FTPClient.isConnected()' on a null object reference
        at net.gotev.uploadservice.ftp.FTPUploadTask.upload(FTPUploadTask.java:139)
        at net.gotev.uploadservice.UploadTask.run(UploadTask.java:162)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:764)
```

Server side
FTP Server on Direct Admin 1.57.5

**Comment by gotev** [comment]
Thanks for this issue! NPEs should be gone after the migration to Kotlin which is in progress #450 and which will be released soon in 4.0.0

**Comment by gotev** [comment]
@yotewach yotewach Upload Service 4.0.0 Alpha 01, 100% Kotlin with Android 10 support is here for developer preview! [Check this issue for more information](https://github.com/gotev/android-upload-service/issues/462). FTP Module has been migrated fully to Kotlin. Try it and tell me if your issue is solved.
